includes/Abilities/Image/Generate_Image.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Image generation WordPress Ability implementation.
+ *
+ * @package WordPress\AI
+ */
+
+declare( strict_types=1 );
+
+namespace WordPress\AI\Abilities\Image;
+
+use WP_Error;
+use WordPress\AI\Abstracts\Abstract_Ability;
+use WordPress\AI_Client\AI_Client;
+use WordPress\AiClient\Files\Enums\FileTypeEnum;
+
+use function WordPress\AI\get_preferred_image_models;
+
+/**
+ * Image generation WordPress Ability.
+ *
+ * @since x.x.x
```

Comment:
Just a note: In `WordPress/performance`, `WordPress/php-ai-client`, and a few other projects we established the convention of `@since n.e.x.t`. Obviously it's an arbitrary decision, but I wonder whether we should use that here too.

Most importantly, we should probably choose a convention and document it, so that it's easy for a pre-release step to replace all those with the actual version. Maybe we can even build linting for it at some point.

This is mostly a thing we'll want to figure out soon, feel free to change it here or leave as is if you prefer.

Reply:
I'm fine if we update to using this, no strong opinion on my end but noting this is [currently documented](https://github.com/WordPress/ai/blob/develop/CONTRIBUTING.md#general-rules) as the approach (which is why I used that here) and there is a step in [our release process](https://github.com/WordPress/ai/blob/develop/CONTRIBUTING.md#release-instructions) to update these